cli/status/status.go

```diff
@@ -1,57 +1,185 @@
 package status
 
 import (
+	_ "embed"
+	"fmt"
 	"os"
+	"strings"
 
+	"github.com/fatih/color"
 	"github.com/jedib0t/go-pretty/v6/table"
 	"github.com/jedib0t/go-pretty/v6/text"
+	"github.com/mitchellh/mapstructure"
 	"github.com/tarantool/tt/cli/connector"
 	"github.com/tarantool/tt/cli/process_utils"
 	"github.com/tarantool/tt/cli/running"
 )
 
+//go:embed lua/instance_state.lua
+var replicationInfoLuaScript string
```

Comment:
Nit: Shouldn't the variable have some more general name? It is not only about the replication info.

Reply:
replaced `replication` with `instance`.